src/core/rtkQuery/splitV3Api/seriesApi.ts

```diff
@@ -149,7 +149,7 @@ const seriesApi = splitV3Api.injectEndpoints({
     }),
 
     // Gets anidb recommendation for the user
-    getAniDBRecommendedAnime: build.query<SeriesRecommendedType[], PaginationType>({
+    getAniDBRecommendedAnime: build.query<SeriesRecommendedType[], RecommendedPaginationType>({
```

Comment:
```suggestion
    getAniDBRecommendedAnime: build.query<SeriesRecommendedType[], PaginationType & { includeRestricted: boolean; }>({
```

I don't see a point on adding a whole another type if it's not going to be used elsewhere. This should be just fine. Remove the type declaration

Reply:
Applied the requested changes